htk-jenkins/entrypoint

```diff
@@ -5,11 +5,11 @@ if [ ! -s /var/jenkins_home/config.xml ];then
 	cp -r /var/jenkins_home_default/./ /var/jenkins_home/
 fi
 export JAVA_OPTS+=" -Dorg.csanchez.jenkins.plugins.kubernetes.pipeline.ContainerExecDecorator.websocketConnectionTimeout=180"
-export JAVA_OPTS+=" -Xloggc:$JENKINS_HOME/gc-%t.log -XX:NumberOfGCLogFiles=5 -XX:+UseGCLogFileRotation -XX:GCLogFileSize=20m -XX:+PrintGC -XX:+PrintGCDateStamps -XX:+PrintGCDetails -XX:+PrintHeapAtGC -XX:+PrintGCCause -XX:+PrintTenuringDistribution -XX:+PrintReferenceGC -XX:+PrintAdaptiveSizePolicy"
```

Comment:
cool; one thing to note around here: the `org.csanchez.` stuff above this is there because "sometimes" there were timeout via during pod executions - I hope those will go away with the newer image/etc :D